pkg/didcomm/crypto/legacy/authcrypt/decrypt.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package authcrypt
+
+import (
+	"golang.org/x/crypto/chacha20poly1305"
+	"golang.org/x/crypto/nacl/box"
+	errors "golang.org/x/xerrors"
+)
+
+// Decrypt will decode the envelope using the legacy format
+// Using (X)Chacha20 encryption algorithm and Poly1035 authenticator
+func (c *Crypter) Decrypt(envelope []byte, recipientPrivKey *[chacha20poly1305.KeySize]byte) ([]byte, error) {
+
+	return nil, nil
```

Comment:
Is there a missing TODO?

Reply:
Added, not marking as outdated for some reason

Reply:
#294